apps/playground/src/app/modules/child/child.module.ts

```diff
@@ -0,0 +1,31 @@
+import {NgModule} from "@angular/core";
+import {ChildComponent} from "./child.component";
+import {CommonModule} from "@angular/common";
+import {Ui5MainModule} from "@ui5/webcomponents-ngx";
+import {RouterModule} from "@angular/router";
+import {Ui5I18nModule} from "@ui5/webcomponents-ngx/i18n";
+import {HttpClientModule} from "@angular/common/http";
+
+@NgModule({
+  declarations: [ChildComponent],
+  imports: [
+    CommonModule,
+    Ui5MainModule,
+    HttpClientModule,
+    Ui5I18nModule.forChild({
+      name: 'i18n_child',
+      translations: {
+        en: fetch('assets/i18n/child/messages_en').then(r => r.text()),
```

Comment:
Does this one supports `HttpClient` case? Maybe some example would be nice

Reply:
Yeah, it supports. Documentation will cover that, but meanwhile in app.module it is added I think